packages/next/client/image.tsx

```diff
@@ -353,6 +375,12 @@ export default function Image({
     delete rest['layout']
   }
 
+  if (layout === 'raw' && !experimentalLayoutRaw) {
```

Comment:
Lets move this check down to where the other errors are thrown. That way it only throws for `next dev` and can be removed as dead code for production builds.

Reply:
Done